kotlin/src/test/kotlin/org/axonframework/extensions/kotlin/serializer/MetaDataSerializerTest.kt

```diff
@@ -0,0 +1,228 @@
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.cbor.Cbor
+import kotlinx.serialization.json.Json
+import org.axonframework.extensions.kotlin.serialization.AxonSerializersModule
+import org.axonframework.extensions.kotlin.serialization.KotlinSerializer
+import org.axonframework.messaging.MetaData
+import org.axonframework.serialization.SerializationException
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.time.Instant
+import java.util.*
+
+class MetaDataSerializerTest {
```

Comment:
Missing testcases for what happens if:
- You put a complex type in (exception)
- What happens during deserialization if the JSON if wrongly formatted